README.md

```diff
@@ -107,3 +107,32 @@ Meeting Agenda is maintained [here](https://github.com/redhat-developer/service-
 Please file bug reports on [Github](https://github.com/redhat-developer/service-binding-operator/issues/new). For any other questions, reach out on `service-binding-support@redhat.com`.
 
 
+## Roadmap
+
+
+**Key**
+
+⚪️ Concept validated
+
+:white_check_mark: Works, has tests and at least one documented scenario
+
+
+
+### Maturity matrix
+
+| Feature                                        | Alpha                | Beta | GA |
+| -----------------------------------------------| -------------------- | ---- | -- |
+| Binding with Kubernetes resources              |  :white_check_mark:  |      |    |
+| Binding with CRD-backed services               |  :white_check_mark:  |      |    |
+| Binding with OLM CRD-backed services           |  :white_check_mark:  |      |    |
+| Binding with secrets/configmaps                |  :white_check_mark:  |      |    |
+| Binding of PodSpec-based workloads             |  :white_check_mark:  |      |    |
+| Binding of non-PodSpec-based workloads         |  ⚪️    |      |    |
+| Binding with multiple-backing services         |  :white_check_mark:  |      |    |
+| Custom binding variables                       |  :white_check_mark:  |      |    |
+| Auto-detect binding resources                  |  :white_check_mark:  |      |    |
+| Security: Validating admission webhook         |   ⚪️             |      |    |
```

Comment:
not sure if this could be called "feature" - IMHO it sounds as the implementation approach.

IMHO, this table should contain features - functionalities visible to users. Also, an entry should not be added to this table before it lands in some format in the code base.